slices/procps.yaml

```
@@ -0,0 +1,49 @@
+package: procps
+
+essential:
+  - procps_copyright
+
+slices:
+  bins:
+    essential:
+      - init-system-helpers_bins
+      - libc6_libs
+      - libncursesw6_libs
+      - libproc2-0_libs
+      - libsystemd0_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/free:
+      /usr/bin/kill:
+      /usr/bin/pgrep:
+      /usr/bin/pidwait:
+      /usr/bin/pkill: {symlink: pgrep}
```

Where is this coming from?

---

```
root@94af8933277f:/# dpkg -c procps_2%3a4.0.4-4ubuntu3_amd64.deb  | grep bin/pkill
lrwxrwxrwx root/root         0 2024-03-31 08:16 ./usr/bin/pkill -> pgrep
```

---

Oh right, missed that, sorry. I suggest the following format:
```suggestion
      /usr/bin/pkill:
```
Since it's already a symlink, we do not need to manually create one ourselves.